tests/unit/data-viewer.spec.ts

```diff
@@ -0,0 +1,191 @@
+import { shallowMount } from '@vue/test-utils';
+import DataViewer from '../../src/components/DataViewer.vue';
+
+describe('Data Viewer', () => {
+  it('should instantiate', () => {
+    const wrapper = shallowMount(DataViewer);
+
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should display a message when no data', () => {
+    const wrapper = shallowMount(DataViewer, {
+      propsData: {
+        dataset: [],
+      },
+    });
+
+    expect(wrapper.text()).toEqual('No Data Available');
```

Comment:
No capitalization needed in this message I think

Reply:
Why ?

Reply:
https://www.grammarbook.com/punctuation/capital.asp